Dockerfile

```diff
@@ -20,20 +20,34 @@ RUN apt-get update
 RUN apt-get dist-upgrade -y
 
 # Installing packages
-RUN apt-get -y install software-properties-common
-RUN apt-get -y install bzip2 unzip openssh-client git curl zip wget
-RUN apt-get -y install lib32stdc++6 lib32z1 lib32ncurses5 lib32bz2-1.0 --no-install-recommends
-RUN apt-get -y install libxslt-dev libxml2-dev
-RUN apt-get -y install build-essential
+RUN apt-get -y install \
+  build-essential \
+  bzip2 \
+  curl \
+  git \
+  lib32stdc++6 \
+  lib32z1 \
+  lib32ncurses5 \
+  lib32bz2-1.0 \
+  libxslt-dev \
+  libxml2-dev \
+  openssh-client \
+  software-properties-common \
+  unzip \
+  wget \
+  zip \
+  --no-install-recommends
 
-# Update apt
+# Install Java
 RUN apt-add-repository ppa:openjdk-r/ppa
 RUN apt-get update
-
-# Install Java
 RUN apt-get -y install openjdk-8-jdk
```

Comment:
I'm not really sure how licensing works if we pull in the apt package. Can you reach out to our open source team? For now, seems like sticking with OpenJDK is the right thing to do in the context of this PR